documentation/bolt_configuration_reference.md.erb

```diff
@@ -121,11 +121,19 @@ Transport configuration options can be set in both the configuration file and in
 <% @transports[:options].each do |transport, options| %>
 ### <%= transport %>
 
+<% if transport == 'ssh' %>
+| Option | Description | External SSH | Type | Default |
+| ------ | ----------- | ------------ | ---- | ------- |
+<% options.each do |option, data| -%>
+  | `<%= option %>` | <%= data[:desc] %> | <%= data[:external] == true ? 'X' : '' %> | <%= data[:type] == TrueClass ? 'Boolean' : data[:type] %> | <%= @transports[:defaults][transport][option].to_s || "" %> |
```

Comment:
```suggestion
  | `<%= option %>` | <%= data[:desc] %> | <%= data[:external] == true ? '&#10004;' : '' %> | <%= data[:type] == TrueClass ? 'Boolean' : data[:type] %> | <%= @transports[:defaults][transport][option].to_s || "" %> |
```

An 'X' seems a little confusing since it's unclear if it means the option is or is not available. I think using a symbol that's more affirmative might be clearer?